README.md

```diff
@@ -10,6 +10,10 @@ The JavaDocs for this library are available on [GitHub Pages](https://iopipe.git
 
 # Installation & usage
 
+This documentation mostly refers to using Maven for your project, if you
+are using Gradle then you can use the example located at:
+<https://github.com/iopipe/examples/blob/master/java/build.gradle>.
```

Comment:
This can be a markdown link:
`if you are using Gradle, [there's an example build.gradle available](https://github.com/iopipe/examples/blob/master/java/build.gradle):`